profiler/src/main/java/com/splunk/opentelemetry/profiler/Configuration.java

```diff
@@ -82,7 +83,6 @@ Map<String, String> defaultProperties() {
     config.put(CONFIG_KEY_MEMORY_ENABLED, String.valueOf(DEFAULT_MEMORY_ENABLED));
     config.put(CONFIG_KEY_MEMORY_EVENT_RATE, DEFAULT_MEMORY_EVENT_RATE);
     config.put(CONFIG_KEY_CALL_STACK_INTERVAL, DEFAULT_CALL_STACK_INTERVAL.toMillis() + "ms");
-    config.put(CONFIG_KEY_OTLP_PROTOCOL, "http/protobuf");
```

Comment:
This makes it a little more challenging for a casual reader to know the default, but I think it's fine.